helper/chart_data.js

```diff
@@ -2263,3 +2263,255 @@ const createSolutionsArray = async function (data) {
         console.log(err);
     }
 }
+
+
+//Report for entity level report in assessments
+exports.entityLevelReportData = async function (data) {
+
+    return new Promise(async function (resolve, reject) {
+
+        //group the data based on completed date   
+        let groupedSubmissionData = await groupArrayByGivenField(data, "completedDate");
+
+        let completedDateArray = Object.keys(groupedSubmissionData);
+
+        //Sort the date in ascending order
+        completedDateArray = completedDateArray.sort();
+
+        if (completedDateArray.length > config.druid.no_of_assessment_submissions_threshold || 3) {
```

Comment:
@deepakr123 - Use constant for defining the default value, check for a valid value in config json i.e. only number is allowed

Reply:
resolved